index.js

```diff
@@ -212,7 +220,15 @@ NYC.prototype._maybeInstrumentSource = function (code, filename, relFile) {
     return null
   }
 
-  return this.transform(code, {filename: filename, relFile: relFile})
+  var ext, transform
+  for (ext in this.transforms) {
```

Comment:
I agree - it's more concise in the code to just use `for...in`.  And in most cases I would think we would have 2 or 3 extensions to iterate through at the most.  Yes it could be many files, but still I think it has to get into the 100s of thousands before any noticeable difference between `for..in` and `for`.